app/konnect/service-catalog/integrations/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Service Catalog Integrations
+subtitle: Integrate third-party applications into your Konnect organization
+content-type: reference
+---
+
+Integrations are applications - either {{site.konnect_short_name}} internal applications or external 
```

Comment:
Should this say "Konnect-internal" with a hyphen? I noticed that there is a hyphen in the screenshot below, but there isn't one here. Should probably be consistent.

<img width="2032" alt="Screenshot 2024-09-10 at 4 18 07 PM" src="https://github.com/user-attachments/assets/1d27e770-caca-4f5d-b88f-5cd56a6d3caf">